src/psacheckbox/psacheckbox.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from '@mui/material/Checkbox';
+
+export const PSACheckbox = ({
+  checked,
+  onChange,
+  name,
+  color = 'primary',
+  style,
+  dataTest,
+  comparisonKeys,
+  valueKey,
+  dispatchRedux,
+  updateAction,
+}) => {
+  const handleCheckboxChange = () => {
```

Comment:
Can you also remove this function? Because we should send them to the component as props